### PR TITLE
Add time sheet

### PR DESCRIPTION
**Pull request by elijah-parrish**
Added the toggle time sheet report as a png, also added the team roles to the list of team members

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -15,11 +15,11 @@ function App() {
       <h3>The Team</h3>
       <h4>Seniors</h4>
       <ul>
-        <li>Brianna Fehring</li>
-        <li>Jacob Jirinec</li>
-        <li>Elijah Parish</li>
-        <li>Matt Robison</li>
-        <li>CJ Schaefer</li>
+        <li>Brianna Fehring - Web and Mobile Teck Lead</li>
```
Comment (fehbrize) [inline src/App.js:18]: Typo in "tech"

**Review by fehbrize — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (elijah-parrish) [reply src/App.js]: Resolved!

**Review by cjs8487 — CHANGES_REQUESTED** [review]
Timesheet image name doesn't match the name in `timesheets.json` and doesn't display as a result

**Review by fehbrize — APPROVED** [review]

**Review by cjs8487 — APPROVED** [review]

**Comment by matedro** [comment]
@elijah-parrish please merge so this can be deployed

**Pull request merged**